src/satellite_images_nso/api/nso_georegion.py

```diff
@@ -39,6 +40,30 @@ def correct_file_path(path):
     return path
 
 
+# Function to merge two GeoTIFF files
+@staticmethod
+def merge_tiffs(input_files, output_file):
+
+    raster_to_mosiac = []
+    # Open the input files
+    src_files_to_mosaic = [rasterio.open(file) for file in input_files]
```

Comment:
Do not forget to close these files! Closing them is better for garbage collection & memory management.

Reply:
should be fixed